mission_systems/shooter/nodes/Sabertooth2x12.py

```diff
@@ -11,6 +11,9 @@ def __init__(self,filename,sim=False):
         self.sim = sim
         if not self.sim:
             self.ser = serial.Serial(filename,baudrate=19200)
+        else:
+            global rospy
+            import rospy       
```

Comment:
Why does this only get imported if it's sim? You just need service calls and what not to be skipped in sim mode.